Topic1/key1/test.java

```diff
@@ -0,0 +1,36 @@
+package Topic1.key1;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.ArrayList;
+import java.util.List;
+
+import Topic1.key1.EcommerceWebPage.payMethod;
+
+
+public class test {
+
+	public static void main(String[] args) {
+		
+		EcommerceWebPage web=new EcommerceWebPage(DataConnection.retrieveOffers());
+		
+		web.listItems();
+		web.listOffers();
+		
+		Customer newCustomer1=new Customer("jordan", new CreditCard("jordan", 12, "100.00"),
+				new PaypalAccount("jordan", "123", "", "100.00"), "100.00");
+		
+
+		
+		newCustomer1.addProduct(web.searchProduct("Oferta del Bootcamp"), 1);
```

Comment:
Let's remove that behavior from Customer and use EcommerceWebPage(EcommerceService) as entry point i.e. ecommerService.addProductToCart("productId", "cartId"). It looks like a better place to have the non Customer related behavior I pointed out.